common/djangoapps/contentserver/test/test_contentserver.py

```diff
@@ -56,11 +62,13 @@ def setUpClass(cls):
         # A locked asset
         cls.locked_asset = cls.course_key.make_asset_key('asset', 'sample_static.txt')
         cls.url_locked = unicode(cls.locked_asset)
+        cls.url_locked_versioned = get_versioned_asset_url(unicode(cls.locked_asset))
```

Comment:
Even further comments: this already happens (if there's a digest, it's added regardless of locked vs unlocked.  I'll go ahead and add the appropriate test to utilize `url_locked_versioned`. :)